mandatory/3-function-output.js

```diff
@@ -1,14 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
 function getRandomNumber() {
+  // we use Random  method to get a random number after that multplie this number 10 
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
 function combine2Words(word1, word2) {
+  //The concat() method joins two or more strings.
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord+" "+secondWord+" "+thirdWord;
```

Comment:
It will be really readable and nicer if you can add spaces between your strings and variables here like so:
`return firstWord + " " + secondWord + " " + thirdWord; `